### PR TITLE
obal was quietly setting an env variable that debug output didn't show

### PR DESCRIPTION
**Pull request by komidore64**

### CODE REVIEW

File: obal/__init__.py [hunk 1]
```diff
@@ -324,7 +324,7 @@ def main(cliargs=None):  # pylint: disable=R0914
     ansible_playbook = (["ansible-playbook"] + ansible_args)
 
     if args.verbose:
-        print(ansible_playbook)
+        print("ANSIBLE_CONFIG={} {}".format(os.environ["ANSIBLE_CONFIG"], ' '.join(ansible_playbook)))
```
Comment (ekohl) [inline obal/__init__.py]: This is the same, but wondering if it's better.
```suggestion
        print("ANSIBLE_CONFIG={}".format(os.environ["ANSIBLE_CONFIG"]), ' '.join(ansible_playbook))
```

**Comment by komidore64** [comment]
PR title had to stick with the obfuscated theme :stuck_out_tongue:

File: obal/__init__.py (hunk 1 above)
Comment (ekohl) [reply obal/__init__.py]: Note that I think this is just enough to get you under a 100 characters and make pylint happy.

Comment (komidore64) [reply obal/__init__.py]: Awesome!

**Comment by komidore64** [comment]
I fixed Travis's `ansible-lint` problem in #182 by pinning `PyYaml` to `< 5.1`.

**Pull request merged**